scenarioo-client/test/protractorE2E/specs/list_scenarios.js

```diff
@@ -35,4 +40,43 @@ useCase('List scenarios')
                 scenarioPage.expectOnlyCollapseAllButtonIsDisplayed();
             });
 
+        scenario('Display Diff-Information')
+            .labels(['diff-viewer'])
+            .it(function () {
+                homePage.goToPage();
+                step('display usecases on homepage');
+                homePage.assertPageIsDisplayed();
+                homePage.chooseComparison(COMPARISON_PROJECTSTART);
+                step('To Projectstart comparison selected');
+                homePage.selectUseCase(SECOND_USE_CASE);
+                step('Use Case selected');
+
+                useCasePage.assertNumberOfDiffInfos(NUMBER_OF_SCENARIOS);
+
+                // Reset
+                homePage.chooseComparison(COMPARISON_DISABLE);
```

Comment:
Es gibt bereits einen besseren Weg solchen State am Anfang für alle Tests immer garantiert richtig zu resetten, siehe dazu die Methode HomePage.initLocalStorage() welche als BeforeEach in allen Tests (ausser in jenen welche explizit mit einem anderen speziellen State testen wollen) aufgerufen wird. in dieser Methode sollten wir auch sicherstellen dass normalerweise keine Comparison für die Tests ausgewählt ist, anstatt es am Ende des Tests immer zu disablen.